src/rust/grapl-web-ui/tests/api/plugin/create.rs

```diff
@@ -0,0 +1,76 @@
+use grapl_web_ui::routes::api::plugin::{
+    create::CreateResponse,
+    get_metadata::GetPluginMetadataResponse,
+};
+
+#[actix_web::test]
+async fn plugin_create() -> eyre::Result<()> {
+    let app = crate::test_app::TestApp::init().await?;
+
+    app.login_with_test_user().await?;
+
+    let plugin_name = "Test Plugin";
+    let create_metadata_body = serde_json::json!({
+            "plugin_name": plugin_name,
+            "plugin_type": "generator",
+            "event_source_id": "00000000-0000-0000-0000-000000000000"
+    });
+    let form = reqwest::multipart::Form::new()
+        .part(
+            "metadata",
+            reqwest::multipart::Part::text(create_metadata_body.to_string()),
+        )
+        .part(
+            "plugin_artifact",
+            reqwest::multipart::Part::bytes("junk".as_bytes()),
+        );
```

Comment:
if you really wanted to exercise it, you _could_ do the `get_sysmon_generator()` - we may want to make a test_fixtures crate to share this everywhere

Reply:
that fn exists already btw

Reply:
hmm yeah. I agree we should probably do that, though I'm thinking in doing so I should expose the `deploy` and `tear_down` APIs as well. I think I'll do that.

Reply:
Kk, I'm not just calling into `e2e-tests::test_fixtures::get_sysmon_generator` for now.